python_src/src/lamp_py/runtime_utils/alembic_migration.py

```diff
@@ -0,0 +1,51 @@
+import subprocess
+import logging
+
+from lamp_py.postgres.postgres_utils import DatabaseManager
+
+
+def run_alembic_migration(db_name: str) -> None:
+    """
+    run alembic migration command at application startup
+    """
+
+    if db_name == "performance_manager":
+        # if database is clean, call to DatabaseManager will trigger
+        # SqlBase.metadata.create_all(self.engine)
+        # creating all database tables
+        db_manager = DatabaseManager()
+    else:
+        raise NotImplementedError(f"Migration for {db_name} not implemented.")
+
+    # check if alembic_version table exists in rds
+    try:
+        db_manager.select_as_list("SELECT 1 FROM alembic_version")
+        version_table_exists = True
+    except Exception as _:
+        version_table_exists = False
```

Comment:
This just checks if the "alembic_version" table exists, if it does not, we assume that that RDS is currently in the state matching the "head" revision.

We create the "alembic_version" table and "stamp" it with the existing head revision.